### PR TITLE
Changed "active" to "activated users" in majority of contexts.

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for:
- https://mattermost.atlassian.net/browse/MM-56483
- https://github.com/mattermost/mattermost/pull/25923
- https://github.com/mattermost/mattermost/pull/25921

Note: A small handful of in-product labels remain as "Active Users" via the System Console UI as noted below, and have not been updated via this docs PR.

Site Statistics:
- Daily Active Users
- Monthly Active Users
- Active Users with Posts

Team Statistics:
- Active Users with Posts
- Recent Active Users

### CODE REVIEW

File: source/about/self-hosted-subscriptions.rst [hunk 1]
```diff
@@ -125,26 +125,26 @@ We'll send you an email notice around the end of the quarter reminding you to se
 
 To send this report, take a screenshot of **System Console > Site Statistics** and send in a reply email back to us. 
 
-- Please ensure your screenshot is taken from the top of the page and includes the **Total Active Users** metric. 
+- Please ensure your screenshot is taken from the top of the page and includes the **Total Activated Users** metric.
```
Comment (johndavidlugtu) [inline source/about/self-hosted-subscriptions.rst:128]: I wonder how we communicate the fact that for versions before v9.5, in the System Console it'd still be "Total Active Users"

File: source/configure/site-configuration-settings.rst [hunk 2]
```diff
@@ -426,14 +426,14 @@ Access the following configuration settings in the System Console by going to **
   :systemconsole: Site Configuration > Users and Teams
   :configjson: .TeamSettings.MaxUsersPerTeam
   :environment: MM_TEAMSETTINGS_MAXUSERSPERTEAM
-  :description: The maximum total number of users per team, including active and inactive users. Default is **50** users per team.
+  :description: The maximum total number of users per team, including activated and inactive users. Default is **50** users per team.
```
Comment (johndavidlugtu) [inline source/configure/site-configuration-settings.rst:429]: activated and deactivated users

We'll also need to update mentions of "inactive users" when it actually means "deactivated users"

File: source/getting-started/architecture-overview.rst [hunk 3]
```diff
@@ -182,7 +182,7 @@ The following diagram is a suggested configuration for Highly-Available database
 Load testing
 ~~~~~~~~~~~~
 
-Mattermost Enterprise Edition was `load tested <https://github.com/mattermost/mattermost-load-test>`__ with 60,000 concurrent active users with:
+Mattermost Enterprise Edition was `load tested <https://github.com/mattermost/mattermost-load-test>`__ with 60,000 concurrent activated users with:
```
Comment (johndavidlugtu) [inline source/getting-started/architecture-overview.rst:185]: Can we confirm loadtesting was based on activated users and not of actual active users?

File: source/about/self-hosted-subscriptions.rst (hunk 1 above)
Comment (cwarnermm) [reply source/about/self-hosted-subscriptions.rst:128]: Good catch. I've updated the bullet point to clarify.

File: source/configure/site-configuration-settings.rst (hunk 2 above)
Comment (cwarnermm) [reply source/configure/site-configuration-settings.rst:429]: Thanks!

File: source/getting-started/architecture-overview.rst (hunk 3 above)
Comment (cwarnermm) [reply source/getting-started/architecture-overview.rst:185]: @johndavidlugtu - Not sure we can with certainty. The load testing details documented on this page likely pre-dates me. How would you like to proceed with the "active/activated" terminology on this specific page?

Comment (johndavidlugtu) [reply source/getting-started/architecture-overview.rst:185]: @nab-77 can you provide some guidance on the language here?

Comment (cwarnermm) [inline source/getting-started/architecture-overview.rst:185]:
```suggestion
Mattermost Enterprise Edition was `load tested <https://github.com/mattermost/mattermost-load-test>`__ with 60,000 concurrent active users with:
```

**Review by johndavidlugtu — APPROVED** [review]
Thanks Carrie!

**Pull request merged**